gke/standard/zonal/basic/main.tf

```diff
@@ -0,0 +1,28 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START gke_standard_zonal_basic]
+resource "google_container_cluster" "default" {
+  name               = "gke-standard-zonal"
```

Comment:
this creates a multi-zonal (instead of a single zonal) cluster, is multi-zonal standard zonal?

Reply:
I was semi-patterning after this example:

https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-zonal-cluster
The following command creates a multi-zonal cluster named `example-cluster`, where the cluster control plane is located in the us-central-a zone, and there are three node locations. The cluster is enrolled in the regular release channel.

```
gcloud container clusters create example-cluster \
    --zone us-central1-a \
    --node-locations us-central1-a,us-central1-b,us-central1-c
```

I think I'll split into separate single vs multi similar to regional.

Reply:
@yliaog - Should be clearer now?

Reply:
yes, it is clearer.